### ISSUE TITLE
CSS selector change in PR #2492 hides the checkbox input when editing a Field:Boolean

### ISSUE THREAD

**Issue by jasongorst** [thoughtbot/administrate#2511]
In app/assets/stylesheets/administrate/components/_buttons.scss, the change in PR #2492 from
```
button,
input[type="button"],
input[type="reset"],
input[type="submit"],
.button {
```
to
```
button,
input,
.button {
```
makes these rules apply to all input types, including checkboxes.

**Comment by nickcharlton** [comment]
Oh, oops. 

Do you have any thoughts on what might work, given the linting rules and expected behaviour?

**Comment by jasongorst** [comment]
> Do you have any thoughts on what might work, given the linting rules and expected behaviour?

Just the attribute selectors would work,
```
[type="button"],
[type="reset"],
[type="submit"],
```

**Comment by jasongorst** [comment]
I added a fix in [PR #2512](https://github.com/thoughtbot/administrate/pull/2512).
